src/main/java/org/jabref/logic/importer/fileformat/EndnoteXmlImporter.java

```diff
@@ -121,9 +121,7 @@ public ParserResult importDatabase(BufferedReader reader) throws IOException {
         try {
             Object unmarshalledObject = unmarshallRoot(reader);
 
-            if (unmarshalledObject instanceof Xml) {
-                // Check whether we have an article set, an article, a book article or a book article set
-                Xml root = (Xml) unmarshalledObject;
+            if (unmarshalledObject instanceof Xml root) {
```

Comment:
it deleted the comment

Reply:
I modified the comment and added it to line 123